### DISCUSSION TITLE
Frigate Integration missing? (not listed)

### DISCUSSION THREAD

**Discussion by Lockie85** [blakeblackshear/frigate#7207] [Ask A Question] [answered]
Hey,

I'm trying to use this Home Assistant blueprint:
https://github.com/SgtBatten/HA_blueprints

However, I've an issue where it's not working as I don't think I've setup Frigate as an integration (Addon is installed and working).

My issue is, when I try to add Frigate under:
Settings > Devices & Services > Add Integration
Frigate does not appear.

I've tried clearing cache, incognito mode but still no luck. I even tried a different browser entirely (Edge). Any idea why this might be occurring?

Many thanks

**Comment by NickM-27** [comment]
you need to download the integration per the integration docs

**Reply by Lockie85** [reply]
Do you mean this doc?
https://docs.frigate.video/integrations/home-assistant
As this is what I followed.

**Reply by NickM-27** [reply]
So the integration has been installed via HACS?

**Reply by Lockie85** [reply, marked as answer]
I think I've managed to confuse myself (unsurprisingly :D )

I thought I had already installed the add-on via:
Home Assistant > HACS > Integrations > "Explore & Add Integrations" > Frigate

So I thought I just needed to do this part now:
Home Assistant > Settings > Devices & Services > Add Integration > Frigate

However, the add-on is an entirely different install process (which I forgot). Woops!

Sorry for wasting your time!
